clusterloader2/testing/load/config.yaml

```diff
@@ -62,14 +62,14 @@
 {{$totalSchedulerThroughputPods := MaxInt (MultiplyInt 2 $MIN_PODS_IN_SMALL_CLUSTERS) (MultiplyInt 2 .Nodes)}}
 {{$schedulerThroughputPodsPerDeployment := .Nodes}}
 {{$schedulerThroughputNamespaces := DivideInt $totalSchedulerThroughputPods $schedulerThroughputPodsPerDeployment}}
-{{$schedulerThroughputThreshold := DefaultParam .CL2_SCHEDULER_THROUGHPUT_THRESHOLD 0}}
+{{$schedulerThroughputThreshold := DefaultParam .CL2_SCHEDULER_THROUGHPUT_THRESHOLD 100}}
```

Comment:
We shouldn't override this in test config. This will rollout to all places including kubemarks and presubmits. 

We should add override in test-infra repo for two jobs only: 100nodes and 5k nods instead, like in https://github.com/kubernetes/test-infra/pull/19377

Reply:
Why it will rollout to kubemarks and presubmits? I think it makes sense to have some default thresholds when CL2_ENABLE_DENSITY_TEST is true.

Reply:
Yes, you are right. I forgot that we already guard this by CL2_ENABLE_DENSITY_TEST. Makes sense.